### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by MehrdadLinux**
I create Readme.md file and i fill it  as far as i knew

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
```
Comment (the-dr-lazy) [inline README.md]: I feel this Twitter follow link makes Cascade somehow personal; We want to make it a community-based project. So I suggest removing this badge.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
```
Comment (the-dr-lazy) [inline README.md:2]: Putting a license badge instead of an explicit link would be nice.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
```
Comment (the-dr-lazy) [inline README.md]: Let's make the `cascade` title case.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
```
Comment (the-dr-lazy) [inline README.md]: This badge looks redundant to me.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
+> A project management service just to show that real-world Haskell is awesome.
+
+### 🏠 [Homepage](https://github.com/the-dr-lazy/cascade)
+
+### ✨ [Demo](www.demo.com)
```
Comment (the-dr-lazy) [inline README.md]: Let's remove this link until we have a real demo.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
+> A project management service just to show that real-world Haskell is awesome.
```
Comment (the-dr-lazy) [inline README.md]: It worth to state in the first description that the project is in WIP and request for support.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+### 🏠 [Homepage](https://github.com/the-dr-lazy/cascade)
+
+### ✨ [Demo](www.demo.com)
+
+## Install
+
+```sh
+install command
```
Comment (the-dr-lazy) [inline README.md:15]: So, the plan is to distribute binary for Linux and OS X (and maybe Windows). Until then we just should state the project is under heavy development and the user just can install it via building it from scratch.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+## Usage
+
+```sh
+how to use it
+```
```
Comment (the-dr-lazy) [inline README.md]: We really don't know what is gonna happen to the design of Cascade. So, until then let's remove this section for now.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+## Run tests
+
+```sh
+how to test it
+```
+
+## Author
```
Comment (the-dr-lazy) [inline README.md]: As the project has 3 other collaborators let's make `Author` plural.

File: README.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+👤 **Mohammad Hasani**
+
+* Website: https://the-dr-lazy.github.io/
+* Twitter: [@the\_dr\_lazy\_](https://twitter.com/the\_dr\_lazy\_)
+* Github: [@https:\/\/github.com\/the-dr-lazy](https://github.com/https:\/\/github.com\/the-dr-lazy)
+* LinkedIn: [@https:\/\/www.linkedin.com\/in\/the-dr-lazy](https://linkedin.com/in/https:\/\/www.linkedin.com\/in\/the-dr-lazy)
```
Comment (the-dr-lazy) [inline README.md:32]: We should add the other collaborators of the project too.
For template, we can use the [Authors section of the-dr-lazy/purescript-monarch README](https://github.com/the-dr-lazy/purescript-monarch#authors).

File: README.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check [issues page](https://github.com/the-dr-lazy/cascade/issues). You can also take a look at the [contributing guide](https://github.com/the-dr-lazy/cascade/graphs/contributors).
+
+## Show your support
+
+Give a ⭐️ if this project helped you!
```
Comment (the-dr-lazy) [inline README.md]: As I stated let's move this support request to the description to make it more bold.

File: README.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+
+
+## 📝 License
+
+Copyright © 2021 [Mohammad Hasani](https://github.com/https:\/\/github.com\/the-dr-lazy).
```
Comment (the-dr-lazy) [inline README.md]: For now, this project is under my maintainership but I want to license it under the project name `Cascade`.

File: README.md [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+
+This project is [MPL--2.0 License](https://github.com/the-dr-lazy/cascade/blob/master/LICENSE) licensed.
```
Comment (the-dr-lazy) [inline README.md]: Does it have two dashes (`--`) or it's just a typo?

**Review by the-dr-lazy — CHANGES_REQUESTED** [review]
Hi @MehrdadLinux.
It's nice to see the first PR as documentation PR. Thank you.
I will propose some suggestions for the README file.

File: README.md (hunk 7 above)
Comment (MehrdadLinux) [reply README.md:15]: I do not have enough information about the installation, I just left this section for you to complete

**Review by MehrdadLinux — COMMENTED** [review]
done

**Review by MehrdadLinux — COMMENTED** [review]
done

**Review by the-dr-lazy — APPROVED** [review]
Sorry for such a big latency to accept this.

**Comment by the-dr-lazy** [comment]
I think because the original branch has been deleted I was not able to merge it via this PR. So I merged it manually to the `next` branch.

**Pull request closed without merging**